frontend/src/pages/LoginGovCallback.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useCallback, useEffect } from 'react';
+import { parse } from 'query-string';
+import { useAuthContext } from 'context';
+import { User } from 'types';
+import { useHistory } from 'react-router-dom';
+
+type cbResponse = {
+  token: string;
+  user: User;
+};
+
+export const LoginGovCallback: React.FC = () => {
```

Comment:
Can you add a quick test to verify that login.gov functionality still works (so we can switch back to this if needed)?

Reply:
This page is now fully covered